app/_layouts/extension.html

```diff
@@ -105,18 +132,51 @@
 {% capture plugin_config_for_service %}
 ### Enabling the plugin on a Service
 
-{% if page.params.yaml_examples == false and page.params.k8s_examples == false %}
+{% if page.params.yaml_examples == false and page.params.k8s_examples == false and page.params.manager_examples == false and page.params.konnect_examples == false %}
 
-{{ plugin_config_for_service_admin_api }}
+  {{ plugin_config_for_service_admin_api }}
 
 {% else %}
 
 {% navtabs %}
 {% navtab Admin API %}
 
-{{ plugin_config_for_service_admin_api }}
+  {{ plugin_config_for_service_admin_api }}
+
+  `<service>` is the `id` or `name` of the Service that this plugin
+    configuration will target.
 
 {% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a Service:
+
+1. In Kong Manager, select the workspace.
+2. From the Dashboard, select **Services**.
+3. Click **View** for the Service row.
+4. Scroll down to plugins and click **Add Plugin**.
+5. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
+    </div>
+
+6. If the option is available, set the plugin scope to **Scoped**.
+
+7. Add the Service field with the Service name and ID if it 
+is not already prefilled. 
+
+{% if config_required_fields_konnect == empty %}
+8. Click **Create** to save.
+  {% else %}  
+8. Enter the following required [parameters](#parameters), updating the default/sample values as needed:
```

Comment:
should avoid slash constructions default/sample

Reply:
k will replace with or